core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -202,6 +202,7 @@ oppia.controller('StateInteraction', [
               $scope.stateInteractionIdService = stateInteractionIdService;
               $scope.stateCustomizationArgsService = (
                 stateCustomizationArgsService);
+              $scope.hasCustomizationArgs = true;
```

Comment:
Added e2e case. If the interaction does not have any customization args, a modal opens up informing them that no customization options are available.